Java/corenative/src/main/java/com/breadwallet/corenative/crypto/BRCryptoNetwork.java

```diff
@@ -25,65 +25,29 @@
 
 public class BRCryptoNetwork extends PointerType {
 
-    public static BRCryptoNetwork createAsBtc(String uids, String name, boolean isMainnet) {
-        return new BRCryptoNetwork(
-                CryptoLibraryDirect.cryptoNetworkCreateAsBTC(
-                        uids,
-                        name,
-                        isMainnet ? BRCryptoBoolean.CRYPTO_TRUE : BRCryptoBoolean.CRYPTO_FALSE
-                )
-        );
+    public static BRCryptoNetwork findBuiltin (String uids) {
+        Pointer builtin = CryptoLibraryDirect.cryptoNetworkFindBuiltin (uids);
+        if (null == builtin) throw new RuntimeException("Missed builtin: " + uids);
```

Comment:
Purely if you are interested; for these types of runtime checks I use the Preconditions checks (checkNotNull, checkState, etc.). These manifest as runtime exceptions.

Reply:
It is a bit harsh to fail on a `uids` lookup...  Changed to Optional.